data.js

```diff
@@ -0,0 +1,30 @@
+var mantras = [
```

Comment:
We now have an API for this. Can't wait to use it!

Reply:
YAYAYAY!